test/var_compatibility_test/var_compatibility_suite.xml

```diff
@@ -5,6 +5,8 @@
     <scheme>effr_pre</scheme>
     <scheme>effr_calc</scheme>
     <scheme>effr_post</scheme> 
-    <scheme>effr_diag</scheme>
+    <subcycle loop="1">
```

Comment:
Would it make sense to test with more than one loop iteration?

Reply:
Done, although bare bone. See 1fe9f2e